tests/Couchbase.Extensions.Caching.IntegrationTests/ClusterFixture.cs

```diff
@@ -1,18 +1,24 @@
 ﻿using System.Threading.Tasks;
+using Couchbase.Compression.Snappier;
+using Couchbase.Extensions.DependencyInjection;
 using Couchbase.KeyValue;
 using Microsoft.Extensions.Configuration;
 using Xunit;
 
 namespace Couchbase.Extensions.Caching.IntegrationTests
 {
-    public class ClusterFixture : IAsyncLifetime
+    public class ClusterFixture : IAsyncLifetime, ICouchbaseCacheBucketProvider, ICouchbaseCacheCollectionProvider
     {
         private bool _bucketOpened;
 
         public ClusterOptions ClusterOptions { get; }
 
         public ICluster Cluster { get; private set; }
 
+        public string ScopeName => throw new System.NotImplementedException();
+
```

Comment:
Is this needed?

Reply:
We need to have it there for the fixture due to the ICouchbaseCacheCollectionProvider interface, but it's not used.